aviary/utils/preprocessors.py

```diff
@@ -35,30 +36,115 @@ def preprocess_crewpayload(aviary_options: AviaryValues):
+        aviary_options.set_val(Aircraft.CrewPayload.NUM_FIRST_CLASS,
+                               aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_FIRST_CLASS))
+        aviary_options.set_val(Aircraft.CrewPayload.NUM_BUSINESS_CLASS,
+                               aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_BUSINESS_CLASS))
+        aviary_options.set_val(Aircraft.CrewPayload.NUM_TOURIST_CLASS,
+                               aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_TOURIST_CLASS))
+    # user has not supplied detailed information on design but has supplied summary information on passengers
+    elif design_num_pax != 0 and num_pax == 0:
+        print("INFO: In preprocessor.py: User has specified Design.NUM_PASSENGERS but not how many passengers are on the flight in NUM_PASSENGERS. Assuming they are equal.")
+        aviary_options.set_val(Aircraft.CrewPayload.NUM_PASSENGERS, design_num_pax)
+
+    # Performe checks on the final data tables to ensure Design is always large then As-Flow
+    if aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_FIRST_CLASS) < aviary_options.get_val(Aircraft.CrewPayload.NUM_FIRST_CLASS):
+        raise om.AnalysisError(
+            f"ERROR: In preprocesssors.py: NUM_FIRST_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.NUM_FIRST_CLASS)}) is larger than the number of seats set by Design.NUM_FIRST_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_FIRST_CLASS)}) .")
+    if aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_BUSINESS_CLASS) < aviary_options.get_val(Aircraft.CrewPayload.NUM_BUSINESS_CLASS):
+        raise om.AnalysisError(
+            f"ERROR: In preprocesssors.py: NUM_BUSINESS_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.NUM_BUSINESS_CLASS)}) is larger than the number of seats set by Design.NUM_BUSINESS_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_BUSINESS_CLASS)}) .")
+    if aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_TOURIST_CLASS) < aviary_options.get_val(Aircraft.CrewPayload.NUM_TOURIST_CLASS):
+        raise om.AnalysisError(
+            f"ERROR: In preprocesssors.py: NUM_TOURIST_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.NUM_TOURIST_CLASS)}) is larger than the number of seats set by Design.NUM_TOURIST_CLASS ({aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_TOURIST_CLASS)}) .")
+    if aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_PASSENGERS) < aviary_options.get_val(Aircraft.CrewPayload.NUM_PASSENGERS):
+        raise om.AnalysisError(
+            f"ERROR: In preprocesssors.py: NUM_PASSENGERS ({aviary_options.get_val(Aircraft.CrewPayload.NUM_PASSENGERS)}) is larger than the number of seats set by Design.NUM_PASSENGERS ({aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_PASSENGERS)}) .")
+
+    # dnp = aviary_options.get_val(Aircraft.CrewPayload.Design.NUM_PASSENGERS)
+    # print(f"INFO: In preprocessor.py: Aircraft has been designed for {dnp} passengers.")
 
```

Comment:
I had no plans for that because it does not appear that `Aircraft.CrewPayload.CARGO_MASS` feeds into any other mass calculations other than `CARGO_CONTAINER_MASS` and I don't see any affect of those items towards aircraft design terms like structure mass.